app/bundles/LeadBundle/Controller/AjaxController.php

```diff
@@ -766,11 +766,17 @@ protected function updateLeadFieldValuesAction(Request $request)
         $dataArray = ['success' => 0, 'options' => null, 'operators' => null, 'disabled' => false];
         $leadField = $this->getModel('lead.field')->getRepository()->findOneBy(['alias' => $alias]);
 
-        if ($leadField) {
+        if($alias == 'notifications'){
```

Comment:
Ok. I did sync with Github  desktop and CS fixer wasn't applied. 
Now I found way how commit with CS fix. I will use it in future.